x-pack/test/functional/apps/lens/index.ts

```diff
@@ -5,84 +5,122 @@
  * 2.0.
  */
 
+import { EsArchiver } from '@kbn/es-archiver';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ getService, loadTestFile, getPageObjects }: FtrProviderContext) {
+export default ({ getService, loadTestFile, getPageObjects }: FtrProviderContext) => {
   const browser = getService('browser');
   const log = getService('log');
   const esArchiver = getService('esArchiver');
   const kibanaServer = getService('kibanaServer');
   const PageObjects = getPageObjects(['timePicker']);
+  const config = getService('config');
+  let remoteEsArchiver;
 
   describe('lens app', () => {
+    const localIndexPatternString = 'logstash-*';
+    const remoteIndexPatternString = 'ftr-remote:logstash-*';
+    const localFixtures = {
+      esArchive: 'x-pack/test/functional/es_archives/logstash_functional',
```

Comment:
The esArchive is the same in both cases.  The only difference is in using the esArchiver or the remoteEsArchiver to load it.  So I'm not sure if it makes it more or less confusing the include it here.  Let's see what other reviewers think.